custom_components/tuya_local/devices/woox_r7246_ir_remote_with_th_sensor.yaml

```diff
@@ -13,6 +13,19 @@ entities:
         mapping:
           - scale: 10
         class: measurement
+  - entity: select
+    translation_key: temperature_unit
+    category: config
+    dps:
+      - id: 11
```

Comment:
This may also need to be added to the temperature sensor entity (as "name: unit" with mappings to capital "C" and "F") instead of a hardcoded "C".